.babelrc

```diff
@@ -3,6 +3,15 @@
   "plugins": [],
   "env": {
     "commonjs": {
+      "presets": [
+        [
+          "@babel/preset-env",
+          {
+            "loose": true,
+            "modules": "commonjs"
+          }
+        ]
+      ],
       "plugins": [
```

Comment:
We don't need this or `@babel/plugin-transform-modules-commonjs` dependency if using `preset-env`